languages/common-lisp/reference/exercise-concepts/anagram.md

```diff
@@ -0,0 +1,20 @@
+# Concepts of Anagram
+
+While there are, of course, different ways to attack this problem the
+'canonical' way is to write a function that "normalizes" the
+candidates and the input word so that they can then be easily
+compared. It is this method which was reviewed for concepts. (ref. the
+[exercism v2 common-lisp track exercise][v2-exercise].)
+
+## Concepts
+
+- Sequences
+- Strings
+- Higher-order functions
+- Equality
```

Comment:
The reference equivalent of Equality is ["Sameness"](https://github.com/exercism/v3/blob/master/reference/concepts/sameness.md). I think either works, but consistency is always good to have.